server/lib/logger.js

```diff
@@ -19,14 +19,8 @@ Logger.writeLog = (errorCode, errorMessage, data, email) => {
   logString += 'Date: ' + date + '\r\n'
   logString += 'Error Code: ' + errorCode + '\r\n'
   logString += 'Error Message: ' + errorMessage + '\r\n'
-
```

Comment:
Why is this removed? If you parse an object in here it will now just display as the string [object] and not actually show the object properties from what i understand?